scalableminds/webknossos
# Update Text in Upload Dataset View

https://updateuploaddstext.webknossos.xyz/

### Steps to test:
- Click "upload dataset"
- Toggle the options `recommendWkorgInstance` and `isDatasetConversionDisabled` to check that all combinations look nice
- Check links

### To do
- [x] fix tests: update snapshot misc.e2e.js.snap

### Issues:
- fixes #7760

------
(Please delete unneeded items, merge only when none are left open)
